README.md

```diff
@@ -87,6 +87,37 @@ Ejecting `@dojo/cli-build-webpack` will produce a `config/build-webpack/webpack.
 node_modules/.bin/webpack --config=config/build-webpack/webpack.config.js
 ```
 
+### Interop with Dojo 1 libraries
+
+Dojo 1 libraries, whether built or not, can be included in a Dojo 2 application by configuring certain options in the project's `.dojorc` file.
+`.dojorc` is a JSON file that contains configuration for Dojo CLI tasks. Configuration for the `dojo build` task can be provided under the
+`build-webpack` property. Dojo 1 dependencies can be specified via a property called `externals` within the `build-webpack` config.
+`externals` is an object and the keys are paths, relative to `node_modules`, specifying the directories or files of the Dojo 1 dependencies that should
+be included. Each value can take three possible types:
+* `true`: Use this if none of the other options apply
+* An array listing the package names that are in the dependency. This list is needed in order to tell the build process to leave the resolution of these
+files up to Dojo. If any module from a package is imported within application code but not specified in this list, it will fail at runtime.
+* An object with the following properties:
+ * `packages`: An array of package names, this array serves the same purpose as providing an array as the value.
+ * `hasLoader`: An optional flag to indicate that the specified layer includes a loader and a separate loader is not needed.
+ * `main`: Can be provided if the external dependency is a folder. This is a path within the module to the file to be required. The folder or file
+ itself will be required if this is not specified. This is only relevant if `loadImmediately` or `hasLoader` is `true`.
+ * `to`: An alternate location for a dependency to be copied to. The location is relative to the `externals`
+ folder. This can be useful if, for example, multiple built modules are being used that have shared dependencies, to ensure that only one copy
+ of each is included in the final build.
+ * `loadImmediately`: An optional flag that indicates this dependency should be loaded before the main application layer. The typical use case
+ for this is to load a layer file so that its contained modules will be available to the application. If a dependency has the loader,
+ `loadImmediately` is not needed as the loader script will be executed regardless.
+
+Types for any dependencies included in `externals` can be installed in `node_modules/@types` just like any other dependency.
+
+If the Dojo 1 library in question is a built layer, and has no external dependencies, then the `externals` config is all that's
```

Comment:
then the `externals` config is all that's -> then `externals` is the only config